HandEyeCalibration/Transform/Transform.cpp

```diff
@@ -0,0 +1,330 @@
+        std::cout << "\nConverting rotation matrix to quaternion" << std::endl;
+        Eigen::Quaterniond quaternion = Eigen::Quaterniond(rotationMatrix);
+        Eigen::Vector4d quaternionReordered = quaternion.coeffs();
+        std::cout << "\nQuaternion:\n" << quaternionReordered << std::endl;
+        fileStorage << "Quaternion format"
+                    << "qx qy qz qw ";
+        fileStorage.write("Quaternion", TransformUtils::eigen2cv(quaternionReordered));
+
+        // Rotation Matrix to Roll-Pitch-Yaw
+        const TransformUtils::Rotation_s rotations[4] = {
+            { TransformUtils::Rotation::ZYX_INTRINSIC, "ZYX_INTRINSIC", "RollPitchYawIntrinsicZYX" },
+            { TransformUtils::Rotation::XYZ_EXTRINSIC, "XYZ_EXTRINSIC", "RollPitchYawExtrinsicXYZ" },
+            { TransformUtils::Rotation::XYZ_INTRINSIC, "XYZ_INTRINSIC", "RollPitchYawIntrinsicXYZ" },
+            { TransformUtils::Rotation::ZYX_EXTRINSIC, "ZYX_EXTRINSIC", "RollPitchYawExtrinsicZYX" }
+        };
+        Eigen::Array3d rollPitchYaw[4];
+        for(uint32_t i = 0; i < 4; i++)
+        {
+            std::cout << "\nConverting from rotation matrix to roll-pitch-yaw angles (" << rotations[i].short_desc
+                      << ")" << std::endl;
+            auto temp_rollPitchYaw = TransformUtils::rotmat2rpy(rotationMatrix, rotations[i].rot);
+            std::cout << "\nRollPitchYaw" << rotations[i].short_desc << ":\n" << temp_rollPitchYaw << std::endl;
+            fileStorage.write(rotations[i].rpy_name, TransformUtils::eigen2cv(temp_rollPitchYaw));
+            rollPitchYaw[i] = temp_rollPitchYaw;
+        }
+        fileStorage.release();
+
+        /*************************************************************************************************************
+         *      Direction robot to Zivid (any format --> Rotation Matrix)
+         *************************************************************************************************************/
+
+        // Roll, Pitch and Yaw to Rotation Matrix
+        Eigen::Matrix3d rotationMatrixFromRollPitchYaw[4];
+        for(uint32_t i = 0; i < 4; i++)
+        {
+            std::cout << "\nConverting roll-pitch-yaw angles (" << rotations[i].short_desc << ") to rotation matrix"
+                      << std::endl;
+            Eigen::Matrix3d temp_rotationMatrixFromRollPitchYaw =
+                TransformUtils::rpy2rotmat(rollPitchYaw[i], rotations[i].rot);
+            std::cout << "Rotation Matrix:\n" << temp_rotationMatrixFromRollPitchYaw << std::endl;
+
+            // Calculate error in transformation (Rotation Matrix --> Roll-Pitch-Yaw --> Rotation Matrix)
+            std::cout << "\nVerifying Rotation Matrix from Roll-Pitch-Yaw angles (" << rotations[i].short_desc << ")"
+                      << std::endl;
+            TransformUtils::compareRotation(rotationMatrix, temp_rotationMatrixFromRollPitchYaw);
+            TransformUtils::compareRotationExtended(rollPitchYaw[i],
```

Comment:
We are definitely overdoing it here. The printout on cmd is super confusing. We need to simplify this. Maybe we can even completely remove it. The verification is made for our own convenience so we could check if the conversions work properly. I think it makes no sense for a customer to use this verification and it just adds confusion.

What do you think?

![image](https://user-images.githubusercontent.com/46567886/69254039-e3f9b380-0bb5-11ea-9cbe-bd53495daaef.png)